pkg/apis/apidiscovery/types.go

```diff
@@ -110,6 +110,12 @@ type APIResourceDiscovery struct {
 	// +listType=map
 	// +listMapKey=subresource
 	Subresources []APISubresourceDiscovery
+	// serverIDs is a list of API server identifiers that serve this resource.
+	// This enables clients to understand which servers in a multi-server deployment
+	// are capable of handling requests for this resource type.
+	// +listType=set
+	// +optional
+	ServerIDs []string `json:"serverIds,omitempty"`
```

Comment:
How does this get populated?  Is there a chicken-egg problem in the information served here?  Would it be possible to implement UVIP/MVP without this field?